apis/kcl/generate.k

```diff
@@ -0,0 +1,44 @@
+import file
+import yaml
+
+_composition = {
+    apiVersion = "apiextensions.crossplane.io/v1"
+    kind = "Composition"
+    metadata = {
+        name = "kcl.xnetworks.aws.platform.upbound.io"
+        labels = {
+            provider = "aws"
+            function = "kcl"
+        }
+    }
+    spec = {
+        compositeTypeRef = {
+            apiVersion = "aws.platform.upbound.io/v1alpha1"
+            kind = "XNetwork"
+        }
+        mode = "Pipeline"
+        pipeline = [
+            {
+                step = "kcl"
+                functionRef = {
+                    name = "crossplane-contrib-function-kcl"
+                }
+                input = {
+                    apiVersion = "krm.kcl.dev/v1alpha1"
+                    kind = "KCLRun"
```

Comment:
```suggestion
                    kind = "KCLInput"
```
It looks like it was renamed in the recent function versions https://github.com/crossplane-contrib/function-kcl/pull/129